js_modules/dagster-ui/packages/ui-core/src/pipelines/GraphNotices.tsx

```diff
@@ -84,7 +84,7 @@ export const LoadingNotice: React.FC<{async: boolean; nodeType: 'asset' | 'op'}>
 const LoadingContainer = styled.div`
   background-color: ${Colors.White};
```

Comment:
I could potentially also just remove the background color. For the jobs graph we actually want a top of 40px I think so maybe I just take the top as a prop

Reply:
This looks good! Unrelated to this PR but we should make sure to check that things still look ok on an `op` graph / non-asset job before we ship all this, I think it uses this same component

Reply:
Yeah I posted a screenshot above